src/syntax/errors/list.rs

```diff
@@ -69,17 +70,17 @@ pub fn get_error_desc(err: &ErrorKind) -> (&'static str, String, &'static str) {
         }
         ErrorKind::ForbiddenWhitespace => (
             "E0007",
-            "keyword cannot end with a whitespace".to_owned(),
+            "Keyword cannot end with a whitespace".to_owned(),
             "",
         ),
-        ErrorKind::ForbiddenCallee => (
+        ErrorKind::ForbiddenCaller => (
             "E0008",
-            "a callee has to be a simple identifier".to_owned(),
-            "",
+            "A caller has to be a simple identifier".to_owned(),
+            "Remove `-` from in front of the caller",
```

Comment:
I think `The caller` might be more suitable here. Also, `Remove…` isn't a good advice, if a corresponding message doesn't exist.